doc/non-ansible-configuration-notes.md

```diff
@@ -67,6 +67,45 @@ The jenkins-workspace hosts are setup as standard Node.js nodes but are only giv
 * Extract to `/var`, e.g. so the resulting directory looks like `/var/cov-analysis-linux64-2017.07/` or similar
 * Ensure that the [node-coverity-daily](https://ci.nodejs.org/job/node-daily-coverity/configure) job matches the path used in its explicit `PATH` setting
 
+## Docker hosts
+
+The hosts that run Docker images for "sharedlibs", Alpine Linux and a few other dedicated systems (hosts identified by `grep _docker-x64- inventory.yml`) don't have Docker image reload logic built in to Ansible. Changes to Docker images (adding, deleting, modifying) involve some manual preparation.
+
+The general steps are:
+
+1. Stop the concerned Jenkins systemd service(s)
+2. Disable the concerned Jenkins systemd service(s)
```

Comment:
Could we add the actual commands lines for these in 1 and 2?

Reply:
actually for 1-3 since we have it for 4 and 5 it would make it a no-bainer to follow the instructions.

Reply:
I see it is below, maybe just integrate with this bullet point list, but no strong feeling on that front.